src/Functions.Worker.Extensions.OpenAI/Search/SemanticSearchOutputAttribute.cs

```diff
@@ -0,0 +1,103 @@
+    /// and collection names.
+    /// </summary>
+    /// <param name="connectionName">
+    /// The name of an app setting or environment variable which contains a connection string value.
+    /// </param>
+    /// <param name="collection">The name of the collection or table to search or store.</param>
+    /// <exception cref="ArgumentNullException">
+    /// Thrown if either <paramref name="collection"/> or <paramref name="connectionName"/> are null.
+    /// </exception>
+    public SemanticSearchOutputAttribute(string connectionName, string collection)
+    {
+        this.ConnectionName = connectionName ?? throw new ArgumentNullException(nameof(connectionName));
+        this.Collection = collection ?? throw new ArgumentNullException(nameof(collection));
+    }
+
+    /// <summary>
+    /// Gets or sets the name of an app setting or environment variable which contains a connection string value.
+    /// </summary>
+    /// <remarks>
+    /// This property supports binding expressions.
+    /// </remarks>
+    public string ConnectionName { get; set; }
+
+    /// <summary>
+    /// The name of the collection or table to search.
+    /// </summary>
+    /// <remarks>
+    /// This property supports binding expressions.
+    /// </remarks>
+    public string Collection { get; set; }
+
+    /// <summary>
+    /// The name of the app setting or environment variable containing the required credentials 
+    /// for authenticating with the search provider. See the documentation for the search provider
+    /// extension to know what format the underlying credential value requires.
+    /// </summary>
+    /// <remarks>
+    /// This property supports binding expressions.
+    /// </remarks>
+    public string? CredentialSettingName { get; set; }
+
+
+    /// <summary>
+    /// Gets or sets the ID of the model to use for embeddings.
+    /// The default value is "text-embedding-3-small".
+    /// </summary>
+    /// <remarks>
+    /// This property supports binding expressions.
+    /// </remarks>
+    public string EmbeddingsModel { get; set; } = OpenAIModels.DefaultEmbeddingsModel;
+
+    /// <summary>
+    /// Gets or sets the name of the Large Language Model to invoke for chat responses.
+    /// The default value is "gpt-3.5-turbo".
+    /// </summary>
+    /// <remarks>
+    /// This property supports binding expressions.
+    /// </remarks>
+    public string ChatModel { get; set; } = OpenAIModels.DefaultChatModel;
+
+    /// <summary>
+    /// Gets or sets the system prompt to use for prompting the large language model.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// The system prompt will be appended with knowledge that is fetched as a result of the <see cref="Query"/>.
+    /// The combined prompt will then be sent to the OpenAI Chat API.
+    /// </para><para>
+    /// This property supports binding expressions.
+    /// </para>
+    /// </remarks>
+    public string SystemPrompt { get; set; } = """
```

Comment:
Now that we've broken the in-proc attribute into separate input and output binding attributes, we need to remove the properties that don't apply. For example, `ChatModel` and `SystemPrompt` don't apply for the output binding. Can you go these two binding attributes and remove the properties that aren't relevant to the scenario?

Reply:
`ChatModel` and `SystemPrompt` don't apply for input bindings correct?